apps/admin_api/lib/admin_api/v1/router.ex

```diff
@@ -91,25 +104,28 @@ defmodule AdminAPI.V1.Router do
     post("/api_key.create", APIKeyController, :create)
     post("/api_key.delete", APIKeyController, :delete)
 
+    post("/settings.all", SettingsController, :get_settings)
+
     # Self endpoints (operations on the currently authenticated user)
     post("/me.get", SelfController, :get)
-    post("/me.get_account", SelfController, :get_account)
     post("/me.get_accounts", SelfController, :get_accounts)
+    post("/me.get_account", SelfController, :get_account)
     post("/me.update", SelfController, :update)
+    post("/me.upload_avatar", SelfController, :upload_avatar)
 
-    post("/logout", AuthController, :logout)
+    post("/admin.logout", AdminAuthController, :logout)
   end
 
   # Public endpoints (still protected by API key)
   scope "/", AdminAPI.V1 do
     pipe_through([:api, :client_api])
 
-    post("/login", AuthController, :login)
+    post("/admin.login", AdminAuthController, :login)
```

Comment:
And `/me.login`?

Reply:
Since this is not an authenticated call, it doesn't fall in the same category as all the other `/me.` in which we can expect to have a user assigned in `conn`.